knative-operator/pkg/common/openshift.go

```diff
@@ -18,6 +18,7 @@ var log = Log
 
 func Mutate(ks *servingv1alpha1.KnativeServing, c client.Client) error {
 	stages := []func(*servingv1alpha1.KnativeServing, client.Client) error{
+		ingressClass,
 		egress,
 		ingress,
 		configureIstio,
```

Comment:
I think we need to ditch this now, no? And replace it with `ingressClass`? Feels like this would override Kourier and not use it at all 🤔

Reply:
Yes, you are correct. We can remove the `configureIstio` as it is not used at all. 
Updated.